lib/cog/queries/command.ex

```diff
@@ -43,6 +54,16 @@ defmodule Cog.Queries.Command do
   end
 
   @doc """
+  Retrieve all information about a command
+  """
+  def complete_command(bundle_name, command_name) do
+    named(bundle_name, command_name)
+    |> with_bundle
+    |> with_rules
+    |> with_options
```

Comment:
You can probably nuke these `with_...` funcs and replace it with `|> Repo.preload([:bundle, :rules, [options: :option_type]])` since I think all those queries are run separately whether or not they're part of the query or not.